engine/DJI_E305.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--bldc_engine name="DJI E305"-->
+  <brushless_dc_motor name="DJI E305">
+  <power unit="WATTS"> 320 </power>
```

Comment:
Unles I'm mistaken `power` is not used by `FGBrushlessDCMotor`.

Reply:
Resolved with commit Corrections

Reply:
Resolved with commit Corrections